gpswearos/src/main/java/com/android/gpswearos/presentation/MainActivity.kt

```diff
@@ -0,0 +1,104 @@
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Devices
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.wear.compose.material.Card
+import androidx.wear.compose.material.Text
+
+class MainActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContent {
+            StatusCard(SampleData.statusSample)
+        }
+    }
+}
+
+data class SatelliteStatus(val index: String, val data: String)
+
+@Composable
+fun StatusCard(satStatues: List<SatelliteStatus>) {
+    Card({} , modifier = Modifier
+        .fillMaxWidth()
+        .padding(5.dp)) {
+        Box ( modifier = Modifier
+            .fillMaxSize()
+            .verticalScroll(rememberScrollState()), contentAlignment = Alignment.Center){
+            Row {
+                LabelColumn(satStatues)
+                ValueColumn(satStatues)
+            }
+        }
+    }
+}
+
+@Composable
+fun ValueColumn(satStatues: List<SatelliteStatus>) {
+    Column(
+        modifier = Modifier
+            .wrapContentHeight()
+            .wrapContentWidth()
+            .padding(top = 5.dp, bottom = 5.dp),
+        verticalArrangement = Arrangement.Center,
+        horizontalAlignment = Alignment.Start
+    ) {
+        // Insert break line to leave space
+        Text("\n")
+        for (s in satStatues) {
+            Text(
+                text = s.data,
+                modifier = Modifier.padding(start = 4.dp, end = 4.dp),
+                fontWeight = FontWeight.Bold,
+                fontSize = 13.sp,
+            )
+        }
+        Text("\n")
+    }
+}
+
+@Composable
+fun LabelColumn(satStatues: List<SatelliteStatus>) {
+    Column(
+        modifier = Modifier
+            .wrapContentHeight()
+            .wrapContentWidth()
+            .padding(top = 5.dp, bottom = 5.dp, start = 5.dp, end = 5.dp),
+        verticalArrangement = Arrangement.Center,
+        horizontalAlignment = Alignment.End
+    ) {
+        Text("\n")
+        for (s in satStatues) {
+            Text(
+                text = s.index,
+                modifier = Modifier.padding(start = 4.dp, end = 4.dp),
+                fontWeight = FontWeight.Bold,
+                fontSize = 13.sp,
+            )
+        }
+        Text("\n")
+    }
+}
+
+@Preview(device = Devices.WEAR_OS_SMALL_ROUND, showSystemUi = true)
+@Composable
+fun DefaultPreview() {
+    StatusCard(SampleData.statusSample)
```

Comment:
Looks like this overflows on my preview screen and on the Wear OS Small round API 30 emulator:

![image](https://user-images.githubusercontent.com/928045/195119467-9c2ac298-b0be-4726-9b5b-484f37d4ce3e.png)

Reply:
> Looks like the text overflows each line on my preview screen and on the Wear OS Small round API 30 emulator:
> 
> ![wear-test](https://user-images.githubusercontent.com/928045/195119720-86ce16ac-a663-459c-87b6-462af027970e.png)
> 
> It will probably work better if we combine the label and value into a single string that's centered in the screen.

Ok, thanks. After doing this, I've found that the quotation marks can't be aligned vertically. 
![6MQ6B}8}L_PVU`)X25023~8](https://user-images.githubusercontent.com/69705892/196041650-fa2a4628-80b5-4f2f-8ebb-77f1312bd07f.png)

Does this seem ok?

Reply:
>Ok, thanks. After doing this, I've found that the quotation marks can't be aligned vertically.

Yes, that looks fine to me. I think we need to make some compromises/design changes from the regular mobile version for the form factor of Wear, and this seems reasonable to me.